app/views/places/show.html.haml

```diff
@@ -32,4 +32,28 @@
       %td= @place.rate.to_s
 
 %p
-  = link_to "<-- Back", root_path
\ No newline at end of file
+  = link_to "<-- Back", root_path
+
+
+%h4 Add new review for  this place
+
+.content
+  = render partial: 'reviews/reviews', place: @place
+
+
+
+  %table.table.table-hover
+    %thead
+      %tr
+        %th{:scope => "col"} #
+        %th{:scope => "col"} Author
+        %th{:scope => "col"} Review
+        %th{:scope => "col"} Rate
+    %tbody
+      - Review.all.each_with_index do |review, index|
```

Comment:
You are rendering all reviews not only related to this place

Reply:
Thank, I've changed it